src/lib/services/key-manager.service.ts

```diff
@@ -6,6 +6,8 @@ import { LSP6KeyManager__factory, LSP6KeyManagerInit__factory } from '../..';
 import {
   deployContract,
   deployProxyContract,
+  getContractAddressFromReceipt,
+  getContractAddressFromReceipt$,
```

Comment:
I m very confused why there is one with $ and one without

Reply:
its a RxJS standard, e.g. `const getString = () => 'string'` `const getString$ = () => Observable<'string'>`